scene/main/scene_tree.cpp

```diff
@@ -964,6 +967,31 @@ bool SceneTree::is_paused() const {
 	return paused;
 }
 
+void SceneTree::set_suspend(bool p_enabled) {
+	ERR_FAIL_COND_MSG(!Thread::is_main_thread(), "Suspend can only be set from the main thread.");
+
+	if (p_enabled == suspended) {
+		return;
+	}
+
+	suspended = p_enabled;
+
+#ifndef _3D_DISABLED
+	PhysicsServer3D::get_singleton()->set_active(!p_enabled);
+#endif // _3D_DISABLED
+	PhysicsServer2D::get_singleton()->set_active(!p_enabled);
```

Comment:
There are no checks for pause, so unsuspending will also unpause the servers, while keeping `paused`.